src/index.ts

```diff
@@ -208,6 +212,16 @@ function startApp(appSchema, path: string) {
   })
 
   app.use("/batch", bodyParser.json(), graphqlBatchHTTPWrapper(graphqlServer))
+
+  if (ENABLE_GRAPHQL_UPLOAD) {
```

Comment:
Is this feature flag intending to be used in staging (or prod too)? Either way, nice to have it toggle-able.

Reply:
Yes, that's right. For first iteration this flag will be enable for staging env only